lib/npack.js

```diff
@@ -150,6 +150,38 @@ var syncByCi = function(options, callback) {
 	);
 };
 
+var syncByPreferCi = function(options, callback) {
+	var tmpPkgSubPath;
+
+	Steppy(
+		function() {
+			tmpPkgSubPath = path.join(options.tmpPkgPath, 'package');
+
+			fse.pathExists(
+				path.join(tmpPkgSubPath, 'npm-shrinkwrap.json'),
+				this.slot()
+			);
+
+			npmUtils.getNpmVersion(this.slot());
+		},
+		function(err, shrinkwrapExists, npmVersion) {
+			// `npm ci` becomes available in npm@5.7.0 and higher
+			if (
+				shrinkwrapExists &&
+				semver.gte(npmVersion, '5.7.0')
```

Comment:
I think it's not right to check only minimal version, because in new versions in theory npm may remove ci command. My suggestion to exec `npm ci -h` command and look for exit code.